### PR TITLE
Prevent duplicates when creating labels. 

### PR DESCRIPTION
**Pull request by AlexFalessi**
Also ensure front-end backward compatiblity by returning unique results when fetching labels from DB that have same label name

### CODE REVIEW

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java [hunk 1]
```diff
@@ -2099,7 +2099,7 @@ List<JobLabelInfo> createLabels(List<String> labels)
      */
     @RoleAdmin
     List<JobLabelInfo> setLabels(List<String> labels)
-            throws NotConnectedException, PermissionException, LabelValidationException;
+            throws NotConnectedException, PermissionException, LabelValidationException, LabelConflictException;
```
Comment (fviale) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java:2102]: why not reuse LabelValidationException ?

Comment (AlexFalessi) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java:2102]: It seemed to me that it's not really a label validation exception, which is thrown when there is a formatting issue on a label (special characters).
Should I use the LabelValidationException ?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java [hunk 2]
```diff
@@ -3227,35 +3230,37 @@ private boolean isLabelValid(String label) {
     public List<JobLabelInfo> newLabels(List<String> labels, String username)
             throws LabelConflictException, LabelValidationException {
         List<JobLabelInfo> jobLabelsInfo = new LinkedList<>();
-        for (String label : labels) {
+        Set<String> uniqueLabels = new HashSet<>(labels);
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java:3233]: you should use linkedhashset also here to preserve the order

**Review by fviale — APPROVED** [review]

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java (hunk 2 above)
Comment (AlexFalessi) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java]: OK done.

**Review by aminelouati — APPROVED** [review]

**Pull request merged**